validator/validate_api.py

```diff
@@ -304,6 +316,8 @@ def validation_api_entry():
 
     response = args.get("response", None)
     uid = args.get("uid", None)
+
+    # What is the second arg for?
```

Comment:
default if "uid" not in args. In that case, "arg['uid']" would fail. Please delete.

Reply:
Ah I mean the comment, not the second arg. I restored it.